react/features/invite/components/info-dialog/DialInNumber.web.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { translate } from '../../../base/i18n';
+
+/**
+ * React {@code Component} responsible for displaying a telephone number and
+ * conference ID (pin) for dialing into a conference.
+ *
+ * @extends Component
+ */
+class DialInNumber extends Component {
+    /**
+     * {@code DialInNumber} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The numberic identifier for the current conference, used after
+         * dialing a the number to join the conference.
+         */
+        conferenceID: PropTypes.number,
```

Comment:
Very minor, but I just wonder if we call this conferenceID and then pin, it may sound inconsistent and difficult to read, so we may want to stick to one.

Reply:
This one is tricky because externally it's referred to as a PIN but the api references it conferenceID. I'll stick with conferenceID internally and leave the string to display "PIN:"